src/Core/Settings/GlobalSettings.cs

```diff
@@ -519,4 +522,74 @@ public class DistributedIpRateLimitingSettings
         public int SlidingWindowSeconds { get; set; } = 120;
     }
 
+    public class LogLevelSettings : ILogLevelSettings
```

Comment:
Might be worth splitting these implementations out into a separate file (`LogLevelSettings.cs`?) rather than continuing to tack onto the end of `GlobalSettings`. I guess they do have default values in them, which is nice to see all in one place, but there should a limit to how large this file can grow.